db/migrate/20230117191958_remove_min_and_max_hours_from_work_process.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveMinAndMaxHoursFromWorkProcess < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :work_processes, :minimum_hours
+    remove_column :work_processes, :maximum_hours
+  end
+end
```

Comment:
Let's just delete this migration

Reply:
@jenlu33 can you remove this file too? Otherwise the next migration will break.

Reply:
But looks like you removed `db/migrate/20230117185953_add_default_hours_to_work_process.rb` which should be reinstated.